web-app/src/styled.js

```diff
@@ -0,0 +1,9 @@
+import { injectGlobal } from 'styled-components';
```

Comment:
Are we tied to this injectGlobal term? If we wanted to theme this it'd need changed. Maybe unosquareGlobal or something. Just make it more flexible.

Reply:
We are but it's just styled-components' way of injecting global CSS styles without tying them to any one component.